app/Http/Controllers/QuoteController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\File;
+use League\Csv\Reader;
+
+class QuoteController extends Controller
+{
+    const QUOTE_CHAR_L = '“';
+    const QUOTE_CHAR_R = '”';
+    const DASH_CHAR = ' — ';
+
+    /**
+     * @return Response
+     */
+    public function parse(Request $request)
```

Comment:
Can we change the method name to 'index' to match the RESTful naming system - https://laravel.com/docs/5.1/controllers#restful-resource-controllers